src/shared/database/setup.ts

```diff
@@ -212,10 +212,10 @@ async function setupNetworksTable(): Promise<void> {
   })
 }
 
-async function setupAmendmentsEnabledTable(): Promise<void> {
-  const hasAmendmentsEnabled = await db().schema.hasTable('amendments_enabled')
-  if (!hasAmendmentsEnabled) {
-    await db().schema.createTable('amendments_enabled', (table) => {
+async function setupAmendmentsStatusTable(): Promise<void> {
```

Comment:
Doesn't this table need an `eta` element?

Reply:
I was thinking of using just the date before and compare with `Date.now()` but you are right since there should be a gap between eta and actual flag+1 time. I will work on adding that back.